sofa-boot-project/sofa-boot-core/rpc-sofa-boot/src/main/java/com/alipay/sofa/rpc/boot/container/ServerConfigContainer.java

```diff
@@ -156,6 +164,29 @@ public void startServers() {
             }
         }
 
+        startCustomThreadPoolMonitor();
```

Comment:
Thread pools which create after this period will not be logged.

Reply:
yes, but it seems rarely occur with sofaboot configuration。
if want support this scenario，i suggest creating another task to deal with it.